### ISSUE TITLE
Retail (11.0.2 build 56196): lua errors because GetWatchedFactionData no longer exists.

### ISSUE THREAD

**Issue by Langmans** [Myrroddin/elvui_progressive_databars_colours#1]
See https://warcraft.wiki.gg/wiki/API_C_Reputation.GetWatchedFactionData

I don't think a bugsack report is necessary here :)

I think currentReactionThreshold and nextReactionThreshold are min and max and currentStanding is the value in between:

https://github.com/Gethe/wow-ui-source/blob/live/Interface/AddOns/Blizzard_ActionBar/Mainline/ReputationBar.lua#L77

I'm proposing a quick fix like this (untested because server maintenance a.t.m.):

```lua
local GetWatchedFactionInfo = _G.GetWachedFactionInfo or C_Reputation and C_Reputation.GetWatchedFactionData and function()
    local data = C_Reputation.GetWatchedFactionData()
    return data.name, data.reaction, data.currentReactionThreshold, data.nextReactionThreshold, data.currentStanding, data.factionID
end
```

**Comment by Myrroddin** [comment]
Thank you for the report. I figured this addon would have issues. I've been busy with other things, but will get to it as soon as possible.

**Comment by Myrroddin** [comment]
This should be fixed with version 2.51.
